Documentation/ceph-cluster-crd.md

```diff
@@ -198,6 +198,9 @@ For more details on the mons and when to choose a number other than `3`, see the
   * `modules`: is the list of Ceph manager modules to enable
 * `crashCollector`: The settings for crash collector daemon(s).
   * `disable`: is set to `true`, the crash collector will not run on any node where a Ceph daemon runs
+* `logCollector`: The settings for log collector daemon.
+  * `enabled`: if set to `true`, the log collector will run as a side-car next to each Ceph daemon. Log on file will be turned on and logs will be rotated. (default: false)
+  * `periodicity`: how often to rotate daemon's log. (default: 24h). Specified with a time suffix which may be 'h' for hours or 'd' for days.
```

Comment:
Practically speaking people can set seconds or minutes but I haven't highlighted it in the documentation on purpose. Also, if you rotate too often this will impact the daemon's performance slightly since the signal briefly interrupts the program. I'll add something in that sense in the doc.